front/components/poke/shadcn/ui/data_table_faceted_filter.tsx

```diff
@@ -38,8 +39,8 @@ export function PokeDataTableFacetedFilter<TData, TValue>({
   const selectedValues = new Set(column?.getFilterValue() as string[]);
 
   return (
-    <PokePopover>
-      <PokePopoverTrigger asChild>
+    <PopoverRoot>
+      <PopoverTrigger>
```

Comment:
no `asChild`? If passing it doesn't break the UI, I would advise doing so to prevent a button in a button.

Reply:
My mistake, let's definitely keep it yes

Reply:
Thx for spotting!